content/doc/tutorials/using-jenkins-to-build-a-java-maven-project.adoc

```diff
@@ -0,0 +1,334 @@
+---
+layout: documentation
+title: Using Jenkins to build a Java/Maven project
+section: doc
+---
+
+:toc:
+
+This tutorial shows you how to automate building a simple Java project (built
+with Maven) using Jenkins.
+
+If you are a Java/Maven developer who is new to CI/CD concepts, or you might be
+familiar with these concepts but don't know how to implement building your
+project using Jenkins, then this tutorial is for you.
+
+*Duration:* This tutorial takes 20-40 minutes to complete (assuming you've
+already met the <<prerequisites,prerequisites>> below). The exact duration will
+depend on the speed of your machine and whether or not you've already
+<<download-and-run-jenkins-in-docker,downloaded and run Jenkins in Docker>> from
+link:..[another introductory tutorial].
+
+You can stop this tutorial at any point in time and continue from where you left
+off.
+
+include::doc/tutorials/prerequisites.adoc[]
+
+include::doc/tutorials/download-and-run-jenkins-in-docker.adoc[]
+
+
+=== Fork and clone the sample repository on GitHub
+
+In this tutorial, we'll be building a simple Java application using Maven, whose
+overall build process will be orchestrated in Jenkins. This Java application
+(which you'll obtain from a sample repository on GitHub) simply outputs the
+string "Hello world!" and is accompanied by a unit test to check that the main
+application works as expected.
+
```

Comment:
I think some of this explanation should be merged with the first bit of the introductory content at the top of this page. That way, readers will know up front what they'll be doing before they commence this tutorial.